docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
+
+  Please supply SNS topic name: streamalert-test-alerts
+
+If you look at ``conf/outputs.json``, you'll notice that the SNS topic was automatically added.
+
+6. Configure a rule to send to the alerts topic.
+We will use ``rules/community/cloudtrail/cloudtrail_root_account_usage.py`` as an example, which
+alerts on any usage of the root AWS account. Change the rule decorator to:
+
+.. code-block:: python
+
+  @rule(
+      logs=['cloudwatch:events'],
+      req_subkeys={'detail': ['userIdentity', 'eventType']},
+      outputs=['aws-sns:test-email']  # Add this line
+  )
+  def cloudtrail_root_account_usage(rec):
+
+7. Now we need to update StreamAlert with these changes:
+
+.. code-block:: bash
+
+  # Hook the streamalert-test-data SNS topic up to the StreamAlert rule processor
+  ./manage.py terraform build
+
+  # Deploy a new version of all of the Lambda functions with the updated rule and config files
+  ./manage.py lambda deploy -p all
+
+8. Time to test! Create a file named ``cloudtrail-root.json`` with the following contents:
+
+.. code-block:: json
+
+  {
+    "account": "1234",
+    "detail": {
+      "eventType": "AwsConsoleSignIn",
+      "userIdentity": {
+        "type": "Root"
+      }
+    },
+    "detail-type": "CloudTrail Test",
+    "id": "1234",
+    "region": "us-east-1",
+    "resources": [],
+    "source": "1.1.1.2",
+    "time": "now",
+    "version": "2018"
+  }
+
+This is only a rough approximation of what the real log might look like, but good enough for our purposes.
+Then send it off to the data SNS topic:
+
+.. code-block:: bash
+
+  aws sns publish --topic-arn arn:aws:sns:REGION:ACCOUNTID:streamalert-test-data --message "`cat cloudtrail-root.json`"
+
+If all goes well, an alert should arrive in your inbox within a few minutes!
+If not, look for any errors in the CloudWatch logs for the StreamAlert Lambda functions.
+
+9. After 10 minutes (the default refresh interval), the alert will also be searchable from AWS Athena:
 
-Quick Start
------------
+.. figure:: ../images/alerts-query.png
+  :alt: Query Alerts Table in Athena
+  :align: center
+  :target: _images/alerts-query.png
 
-1. Define your `AWS account <account.html>`_
-2. Define your `clusters <clusters.html>`_
-3. Define your `datasources <conf-datasources.html>`_
-4. Define your `schemas <conf-schemas.html>`_
-5. Configure your `kinesis stream/firehose <kinesis.html>`_
-6. Configure your `lambda settings <lambda.html>`_
-7. Write your `rules <rules.html>`_
-8. Configure your `outputs <outputs.html#configuration>`_
+(Be sure to replace PREFIX with your own name prefix.) If no records are returned, look for errors
+in the ``athena_partition_refresh`` function or try invoking it directly.
 
-Now it's time to `deploy <deployment.html>`_!
+And there you have it! Ingested log data is parsed, classified, and scanned by the StreamAlert rules
+engine and the resulting alert is delivered to your configured output within a matter of minutes.
```

Comment:
`output` --> `output(s)`

Reply:
Wasn't sure whether this sentence should be generic or specific to the test example. I'll go with your suggestion (generic) and reword slightly